WcaOnRails/app/views/regional_organizations/_regional_organization_form.html.erb

```diff
@@ -1,4 +1,4 @@
-<% url = @regional_organization.new_record? ? new_regional_organization_path : edit_regional_organization_path(@regional_organization) %>
+<% url = @regional_organization.new_record? ? new_regional_organization_path : regional_organization_path(@regional_organization) %>
```

Comment:
I think `new_regional_organization_path` also doesn't exist anymore after the Rails 7.2 migration (verify by logging in as admin and clicking "create new" in the "Manage Regional Organizations" panel)

Should probably be replaced by `regional_organizations_path`.

Reply:
Good catch, just pushed the update!